src/test/java/com/study/racing/StringTest.java

```diff
@@ -0,0 +1,53 @@
+package com.study.racing;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("\uD83D\uDE80String 클래스에 대한 학습 테스트 - 1단계")
+class StringTest {
+	String regex = ",";
```

Comment:
어차피 고정된 값이라고 하면 정적변수를 사용하는건 어떨까요?!

```suggestion
	String regex = ",";
	private static final String DELIMITER= ",";
```

Reply:
옙 !! 맞습니다.